.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: python
 matrix:
   include:
-  - python: 3.6
```

Comment:
I actually still run Control Software on Python3.6 on my machine, since ProjectQ gives errors on 3.7 or 3.8. Could you please leave 3.6 in?

Reply:
@olaf @fer Adding Fer to the discussion here. The reason I removed 3.6 is that qcodes stoppped supporting 3.6 (and qtt did as well). We could add 3.6 again to qilib and then probably an old version of qcodes will be retrieved from pypi, but it doesn't feel right to add 3.6 to the CI if qilib/qtt/qi is not supporting this any more. 

If projectq gives error on a particular machine, you can always run an older version of qilib.